File: SistemaProcinco/SistemaProcinco.Common/Models/CursosViewModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SistemaProcinco.Common.Models
{
    public class CursosViewModel
    {
        public int Curso_Id { get; set; }
        public string Curso_Descripcion { get; set; }
        public int? Curso_DuracionHoras { get; set; }
        public string Curso_Imagen { get; set; }
        public int? Cate_Id { get; set; }
        [NotMapped]
        public string Categoria { get; set; }
        public int Curso_UsuarioCreacion { get; set; }
        public DateTime Curso_FechaCreacion { get; set; }
        public int? Curso_UsuarioModificacion { get; set; }
        public DateTime? Curso_FechaModificacion { get; set; }
        [NotMapped]
        public string Creacion { get; set; }
        [NotMapped]
        public string Modificacion { get; set; }
    }
}

File: PlantillaProcinco/src/app/Models/CursosViewModel.ts
import { an } from "@fullcalendar/core/internal-common";
   
   export class Curso {
    curso_Id !: Number;
    curso_Descripcion  !: String;
    curso_DuracionHoras  !: Number;
    curso_Imagen  !: String;
    cate_Id  !: Number;
    categoria  !: String;
    curso_UsuarioCreacion !: Number;
    curso_FechaCreacion !: String;
    curso_UsuarioModificacion !: Number;
    curso_FechaModificacion !: String;
    creacion !: String;
    modificacion !: String;
}

export class dropCursos{
    curso_Id !: Number;
    curso_Descripcion  !: String;
}





// treeviewwwwwwwwwwwwwwwwwwwwwwwwwwww



export class Cursoo{
    curso_Id !: Number;
    curso_Descripcion  !: String;
    curso_DuracionHoras  !: String;
    curso_Imagen  !: String;
    cate_Id  !: String;
    empre_Id  !: String;
    categoria  !: String;
}

export class CursoEnviar {
    Curso_Id?:String;
    txtCurso: string;
    txtxEmpresa: string;
    txtxHoras: string;
    txtImagen: string;
    txtCategoria: string;
    contenidosSeleccionados: any;
}
export class dropCurso{
    value?:String;
    text?:String;
}

export class Fill{
    curso_Id: string;
    cont_Id: string;
    curso_Descripcion  !: String;
    curso_DuracionHoras  !: String;
    curso_Imagen  !: String;
    cate_Id  !: String;
    empre_Id  !: String;
    categoria  !: String;
    usuarioCreacion: string;
    usuarioModificacion: string;
    fechaCreacion : string;
    fechaModificacion : string;
}